implementation/pom.xml

```diff
@@ -131,13 +130,36 @@
                         <SMALLRYE_MP_CONFIG_PROP>1234</SMALLRYE_MP_CONFIG_PROP>
                     </environmentVariables>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <!--
-                    TODO: Confirm if these are necessary
-                    <enableAssertions>true</enableAssertions>
-                    <argLine>-Xmx512m</argLine>
-                    <forkMode>once</forkMode>
-                    -->
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/CustomExtensionParsingTests.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>jackson-exclusion-tests</id>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <includes>
+                                <include>**/CustomExtensionParsingTests.java</include>
+                            </includes>
+                            <systemProperties>
+                                <classpath.jackson.excluded>true</classpath.jackson.excluded>
+                            </systemProperties>
+                            <classpathDependencyExcludes>
+                                <classpathDependencyExclude>com.fasterxml.jackson.core:jackson-core</classpathDependencyExclude>
+                                <classpathDependencyExclude>com.fasterxml.jackson.core:jackson-databind</classpathDependencyExclude>
+                                <classpathDependencyExclude>com.fasterxml.jackson.dataformat:jackson-dataformat-yaml</classpathDependencyExclude>
+                            </classpathDependencyExcludes>
+                        </configuration>
+                    </execution>
+                </executions>
```

Comment:
TIL the surefire plugin is cooler than I realized.

Reply:
Same - I have not used it in this way before. TIL about TIL, also :-D